x/epochs/types/utils.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+import (
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// DurationToBz parses time duration to maintain number-compatible ordering
+func DurationToBz(duration time.Duration) []byte {
+	return sdk.Uint64ToBigEndian(uint64(duration.Milliseconds()))
```

Comment:
why `milliseconds` not `nanoseconds`? I'm worried that we're having inconsistent ways of storing the duration (sec, ms, ns)

Reply:
Done in https://github.com/tharsis/evmos/pull/611